pkg/auth/user/RoleGroupService.go

```diff
@@ -641,7 +643,7 @@ func (impl RoleGroupServiceImpl) FetchRoleGroupsWithFilters(request *bean.FetchL
 	// default values will be used if not provided
 	impl.userCommonService.SetDefaultValuesIfNotPresent(request, true)
 	if request.ShowAll {
-		return impl.FetchRoleGroups()
+		return impl.FetchRoleGroups(request)
```

Comment:
showAll with Filter ?

Reply:
yes we have to respect the filters like search while exporting.